controllers/machinehealthcheck_controller_test.go

```diff
@@ -1582,25 +1551,35 @@ func fakeNodesMachines(
 	return nodes, machines, cleanup
 }
 
-func newTestMachineHealthCheck(name, namespace, cluster string, labels map[string]string) *clusterv1.MachineHealthCheck {
+func newMHCWithLabels(name, namespace, cluster string, labels map[string]string) *clusterv1.MachineHealthCheck {
```

Comment:
```suggestion
func newMachineHealthCheckWithLabels(name, namespace, cluster string, labels map[string]string) *clusterv1.MachineHealthCheck {
```
given that in all the other places we are using the full name